domains/certificates/Query.API/API/DataSyncSyncer/DataSyncService.cs

```diff
@@ -48,6 +48,12 @@ public DataSyncService(IDataSyncClient client, ILogger<DataSyncService> logger,
                     masterData.MeteringPointOwner,
                     cancellationToken
                 );
+                logger.LogInformation(
+                    "Successfully fetched {numberOfMeasurements} measurements in period from {from} to: {to}",
+                    result.Count,
+                        DateTimeOffset.FromUnixTimeSeconds(dateFrom).ToString("o"),
```

Comment:
Can this be indented on the same level as the other arguments?

Reply:
fixed in e961e28